contentcuration/contentcuration/static/js/edit_channel/channel_list/tests/channelEditor.spec.js

```diff
@@ -84,7 +84,7 @@ describe('channelEditor', () => {
       let nameInput = wrapper.find('.channel-name');
       nameInput.element.value = newName;
       nameInput.trigger('input');
-      nameInput.trigger('blur');
+      nameInput.trigger('change');
```

Comment:
This fixes the problem of pressing `Enter` on the editor as it fires before the submit function is emitted. It still keeps the event from firing super often still though as it will trigger once the user exits the textbox